scripts/temp-create-data.js

```diff
@@ -1,9 +1,18 @@
-const { utils, Wallet, Contract, ContractFactory, providers, BigNumber } = require('ethers');
-
-const colonyJSExtras = require('@colony/colony-js/extras')
-const colonyJSIColony = require('../node_modules/@colony/colony-js/dist/cjs/contracts/IColony/9/factories/IColony__factory.js')
-const colonyJSMetaTxToken = require('../node_modules/@colony/colony-js/dist/cjs/contracts/factories/latest/MetaTxToken__factory.js')
-const { addAugmentsB } = require('../node_modules/@colony/colony-js/dist/cjs/clients/Core/augments/AddDomain.js');
+const {
+  utils,
+  Wallet,
+  Contract,
+  ContractFactory,
+  providers,
+  BigNumber,
+} = require('ethers');
+
+const colonyJSIColonyNetwork = require('../node_modules/@colony/colony-js/dist/cjs/contracts/factories/IColonyNetwork__factory.js');
+const colonyJSIColony = require('../node_modules/@colony/colony-js/dist/cjs/contracts/IColony/9/factories/IColony__factory.js');
```

Comment:
These and lines 278-279 and 402 are the only changes. The rest are prettier doing its formatting